docs/sql-guide/functions/function-offset.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OFFSET()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# RTRIM () function
+
+`OFFSET()` function is used in conjuction with the `LIMIT` clause to control the starting point of data retrieval in a SQL query result. It allows skipping a specified number of rows before fetching the desired number of rows.
+
+## Syntax
+
+```
+limit(num_of_rows) Offset(offset_value)
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| `num_of_rows` | An `integer` value specifying the maximum number of rows to fetch. |
+| `offset_value` | An `integer` value specifying the number of rows to skip. |
+
+
+## Returns
+
+The `OFFSET()` function doesn't have a direct return value. It affects the result set returned by the SQL query by skipping the specified number of rows before fetching the data.
+
+
+## Examples
+
+### using OFFSET to skip rows
+
+```sql
```

Comment:
Remove this in favour of the table definition/insert above the examples to avoid duplication.

Reply:
Done